doc/output-details-and-examples/src/output-files/eplusout-eio.tex

```diff
@@ -1019,17 +1019,17 @@ \subsubsection{Field: Outdoor Controlled Baseboard Heat}\label{field-outdoor-con
 
 \subsection{People Gains}\label{people-gains}
 
-! \textless{}People Internal Gains - Nominal\textgreater{},Name,Schedule Name,Zone Name,Zone Floor Area \{m2\},\# Zone Occupants,Number of People \{\},People/Floor Area \{person/m2\},Floor Area per person \{m2/person\},Fraction Radiant,Fraction Convected,Sensible Fraction Calculation,Activity level,ASHRAE 55 Warnings,Carbon Dioxide Generation Rate,Nominal Minimum Number of People,Nominal Maximum Number of People
+! \textless{}People Internal Gains Nominal\textgreater{}, Name, Schedule Name, Zone Name, Zone Floor Area \{m2\}, \# Zone Occupants, Number of People \{\}, People/Floor Area \{person/m2\}, Floor Area per person \{m2/person\}, Fraction Radiant, Fraction Convected, Sensible Fraction Calculation, Activity level, ASHRAE 55 Warnings, Carbon Dioxide Generation Rate, Minimum Number of People for All Day Types, Maximum Number of People for All Day Types, Minimum Number of People for Weekdays,  Maximum Number of People for Weekdays,  Minimum Number of People for Weekends/Holidays,  Maximum Number of People for Weekends /Holidays, Minimum Number of People for Summer Design Days,  Maximum Number of People for Summer Design Days, Minimum Number of People for Winter Design Days,  Maximum Number of People for WinterDesign Days
 
-People Internal Gains,~ SPACE1-1 PEOPLE 1, OCCUPY-1, SPACE1-1, 99.16, 11.0, 11.0, 0.111, 9.015, 0.300, 0.700, AutoCalculate, ACTSCHD, No, 3.8200E-008, 0, 11
+ People Internal Gains Nominal,  SPACE1-1 PEOPLE 1, OCCUPY-1, SPACE1-1, 90.00, 10.0, 10.0, 0.111, 9.009, 0.300, 0.700, 0.550, ACTSCHD, No, 3.8200E-008, 0.0, 10.0, 0.8, 4.0, 0.6, 2.5, 9.5, 10.0, 0.0, 0.5, 
```

Comment:
Noticed here that for all the internal gain tables the word Nominal was added to the table title. That is important enough to note in OutputChanges.

![image](https://github.com/NREL/EnergyPlus/assets/5778156/6a012cfb-e819-4c6b-aa9e-65357c10bc3f)

Reply:
Correction. The word "Nominal" is added to the eio title and was already included in the html using develop. So nothing to do here.